tools/explorer/lib/ontology.py

```diff
@@ -1,12 +1,12 @@
 """Ontology class for ontology explorer."""
```

Comment:
can you please give a more descriptive file name and a class name.

Reply:
renamed to ontology_wrapper and OntologyWrapper respectively